docs/tools/prism-eslint-hook.js

```diff
@@ -317,7 +317,7 @@ function installPrismESLintMarkerHook() {
                     continue;
                 }
 
-                if (Array.isArray(token.content) || typeof token.content !== "string") {
+                if (Array.isArray(token.content)) {
```

Comment:
According to Prism typings, `token.content` can be a string, Token, or array of strings/Tokens, so it looks like this was targeting the Token case, but missed the fact that `token` itself can be a string, and that's causing the crash?

Would it make sense to keep this condition and fix the problem this way:

```diff
for (const token of tokens) {

+    if (typeof token === "string") {
+        yield token;
+        continue;
+   }

    const content = getTokenContent(token);
```

Reply:
The condition is flawed because `Array.isArray(token.content)` implies `typeof token.content !== "string"`, but I think it's a good idea to check if `token` is a string early in the loop nonetheless, because it makes the logic easier to follow. In the below condition it's enough to check that `token.content` is not a string to verify that it's an array. I wasn't able to reproduce the case where `token.content` is a Token, but in theory, it should work as well.